apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.ts

```diff
@@ -1249,10 +1253,10 @@ export class ProformaInvoiceComponent implements OnInit, OnDestroy, AfterViewIni
                 } else {
                     if (result[1]) {
                         result[1] = result[1] as ProformaResponse;
-                        result[1].results.forEach(item => {
+                        result[1].items.forEach(item => {
                             arr.push({
                                 versionNumber: this.isProformaInvoice ? item.proformaNumber : item.estimateNumber,
-                                date: this.isProformaInvoice ? item.proformaDate : item.estimateDate,
+                                date: this.isProformaInvoice ? item.voucherDate : item.voucherDate,
```

Comment:
No need of this.isProformaInvoice ?...
```suggestion
                                date: item.voucherDate,
```